packages/swapper/src/manager/SwapperManager.test.ts

```diff
@@ -1,9 +1,16 @@
 /* eslint-disable @typescript-eslint/ban-ts-comment */
+import Web3 from 'web3'
 import { SwapperType } from '../api'
 import { ThorchainSwapper, ZrxSwapper } from '../swappers'
 import { SwapperManager } from './SwapperManager'
+import { ChainAdapterManager } from '@shapeshiftoss/chain-adapters'
 
 describe('SwapperManager', () => {
+  const zrxSwapperDeps = {
+    web3: ({} as unknown) as Web3,
```

Comment:
This should work as well and looks a little cleaner imo
```suggestion
    web3: <Web3>{},
```